docs/docs/rad-lab-ui/ui_installation/frontend.mdx

```diff
@@ -60,19 +58,50 @@ Add your Project ID here to have the documentation be be customized to your Goog
 
 <ProjectIdInput />
 
-## Add Firebase
+# Add Firebase
+
+:::warning
+If you've never used Firebase in your Google Cloud organization before, please execute [First Time Use](#first-time-use).  If you already have Firebase projects running in your Google Cloud organization or you have in the past, please execute [Returning Firebase User](#returning-firebase-user).
 
-:::important First time
-If you have never used Firebase in your Google Cloud organization, you need to manually add it to the Google Cloud project created in the first step. This is necessary as you have to accept the Terms & Conditions first.
+If you're not sure, simply execute the steps as described in [First Time Use](#first-time-use).
 :::
 
-### Manually
+## First Time Use
 
 Open the [Firebase console](https://console.firebase.google.com) and click on "Create a project" (or add Firebase to an existing project). When prompted with "Enter your project name", enter the project ID you copied from the previous step, and accept the Firebase terms and conditions. Confirm the Firebase billing plan when prompted (and if necessary) and step through the remaining steps in the wizard.
 
 NOTE: Firebase analytics does not impact the deployment of the RAD Lab UI application, so feel free to select whatever you think makes most sense for your deployment.
 
-### Command Line
+Run the following commands in a terminal or command window:
+
+```shell
+# Replace [PROJECT_ID] with the actual project ID.
+export RLUI_PROJECT_ID=$(terraform show -json | jq -r .values.outputs.project_id.value)
+
+# Install Firebase Tools
+npm install -g firebase-tools
+
+# Switch to the webapp directory
+cd ../../../webapp
```

Comment:
It's unclear to me which working directory I should be in when I run `cd ../../../webapp`

Reply:
Normally users can just follow along with these steps (copy/paste).  I'll specify in that comment what directory they should move to, but out of experience I know users don't read that comment :).